index.html

```diff
@@ -13758,6 +13909,8 @@ <h2>Changes since Recommendation of 16 July 2020</h2>
     <li>Misclaneous spelling and grammar fixes.</li>
     <li>Regenerated definition list, which does not attempt to filter out unused definitions.</li>
     <li>Fixed typo in <a href="#alternate-document-location" class="sectionRef"></a> where "HTTP document" was intended to be "HTML document".</li>
+    <li>Allow further structured extensions of `application/ld+json` by using
+      `+ld+json` as a structured media extension. See <a href="#structured-extension-ld-json"></a>.</li>
```

Comment:
@OR13 I pushed this directly to your branch, which I couldn't do through the UI. Feel free to suggest updates to it. Note the changes to the section identifier which are in separate update suggestions.

Reply:
thats fine, I am not attached to any of the specific language here, only to completing the process.... so that it can be taken as a proper dependency of the VC WG and DID WG